src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -235,6 +235,23 @@ public function getEnabledTransitions(object $subject)
         return $enabledTransitions;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getEnabledTransition(object $subject, string $name)
+    {
+        $marking = $this->getMarking($subject);
+
+        foreach ($this->definition->getTransitions() as $transition) {
+            $transitionBlockerList = $this->buildTransitionBlockerListForTransition($subject, $marking, $transition);
```

Comment:
You can do the transition name equal comparison here to avoid useless computation to retreive the good one

Reply:
Done !
Thank you :)